### ISSUE TITLE
Example?

### ISSUE THREAD

**Issue by octaviopardo** [joachimvu/Grader#1]
Hello!
This looks fantastic, but I cannot make it work properly yet. Can you write down an actual example of the syntaxis that we should use in the custom parameter? for example for a font with weight master and italic master.

I am guessing that we should do all the work in the custom parameters window and ignore the grade option from the filters menu, right? I keep getting this message
<type 'exceptions.TypeError'>: runFilterWithLayers:error:: Need tuple of 2 arguments as result
 
Thanks!

**Comment by joachimvu** [comment]
Yes, this has been put together very quickly and the only way for now is by using the custom parameter. It should be written like this: `Grader; pos: 25, 50;` where 25 is the position on your first axis and 50 the position on your second axis. All instances with such a parameter will have the widths and kerning of an instance positioned at the defined coordinates. Let me know if you run into any troubles!

**Comment by octaviopardo** [comment]
Sorry, I have been trying to make it work but I have failed :( This is one of the many options I have been trying. How should I write it?

![Captura de pantalla 2020-09-23 a las 9 41 39](https://user-images.githubusercontent.com/2181684/93982074-10a3cc00-fd81-11ea-8830-ed2eb14e004f.png)

**Comment by joachimvu** [comment]
Sorry, should have been more clear. You need to add a `filter` custom parameter. In the value of the parameter, you need to declare:

1. The name of the filter plugin you want to apply, ending with a semicolon: `Grader;`
2. The name of the option you want to set, in this case `pos`, standing for _position_, followed by a colon and a comma-separated list of values that represent the position of your source for each available axis in your project: `pos:50;`

A correctly defined parameter would look like this:
<img width="363" alt="Capture d’écran 2020-09-23 à 10 35 06 AM" src="https://user-images.githubusercontent.com/1668442/93987953-79427700-fd88-11ea-9e9e-a47941f4e602.png">

**Comment by octaviopardo** [comment]
Ok, now I have been trying several things and all I get is a program crash. I wonder if I have some other plugin or configuration that is uncompatible with your plugin? I will review it again during the weekend more deeply to try to figure out what s failing.

**Comment by joachimvu** [comment]
That may be it, or something in the way your project is set up. Can you send me your Glyphs file? 
I admittedly threw this together very quickly and had no time to test whatsoever. If there is anything expected by the plugin that’s not quite right, I have nothing in place to prevent an error.

**Comment by octaviopardo** [comment]
Hi Joachim! I tried reducing the file to the maximum expression and still got a crash. Here goes! Thanks in advance for your time

[Bris-Test.zip](https://github.com/joachimvu/Grader/files/5290423/Bris-Test.zip)

**Comment by joachimvu** [comment]
Just pushed an update.

Note: You only have to set values for existing axes. If you have one axis, `pos` will have only one value, if you have two axes, you'll need two values (in the order defined in your `axes` parameter in the Font tab).

In the attached zip you'll find two fonts, output from generating with the following parameter:
`Grader;pos:97;` which is a median value between your two instances (handy if you find out that strictly applying one instance's metrics to another creates undesirable spacing)
[Bris_Fonts.zip](https://github.com/joachimvu/Grader/files/5294206/Bris_Fonts.zip)

Two things caused crashes:
1. Glyphs returns values of different types for the custom parameter depending on its content
2. Glyphs makes layers for glyphs that are not in the font but considered essential, but not yet accessible in the instance.
